jwst/pathloss/pathloss.py

```diff
@@ -19,10 +19,36 @@
 NIRSPEC_IFU_SLICES = np.arange(30)
 
 
-def get_center(exp_type, input):
+def get_center(exp_type, input, offsets=False):
     """Get the center of the target in the aperture.
     (0.0, 0.0) is the aperture center.  Coordinates go
     from -0.5 to 0.5.
+
+    Parameters
+    ----------
+    exp_type : str
+        Keyword value
+
+    input_model : data model object
+        science data to be corrected
+
+    offsets: bool
```

Comment:
Here and elsewhere there needs to be a space between the attribute and `:` as in `offset : bool`

Reply:
done